lib/arrow.cpp

```diff
@@ -2039,7 +2077,7 @@ namespace grnarrow {
         auto raw_value_builder = column_builder->value_builder();
         if (grn_obj_is_table_with_key(ctx_, domain)) {
           auto value_builder =
-            static_cast<arrow::StringBuilder *>(raw_value_builder);
+            static_cast<arrow::StringDictionaryBuilder *>(raw_value_builder);
```

Comment:
Originally, reference types except string seem to be not supported at this method because the builder can not handle these types, is it intended behavior at the moment?
Similarly, the not reference string type seems to be not supported.

Reply:
Lacking non `ShortText` reference support is just for no request. :-)

We should support it eventually...

Reply:
I see, thank you!